packages/starlight/expressive-code.mjs

```diff
@@ -15,7 +15,7 @@
 
 export * from 'astro-expressive-code';
 
-// @ts-ignore - Types are provided by the separate `expressive-code.d.ts` file
+// @ts-expect-error - Types are provided by the separate `expressive-code.d.ts` file
```

Comment:
Are we sure there are no scenarios where `expect-error` could cause issues for users here? i.e. in some configuration scenarios, this might no longer be an error causing the `expect-error` check to fail? (e.g. with `noImplicitAny` or `strict` disabled in `tsconfig.json`)

Otherwise could need `// eslint-disable-next-line @typescript-eslint/ban-ts-comment` like in [`packages/starlight/utils/translations.ts`](https://github.com/withastro/starlight/pull/1640/files#diff-2e1c8ef06b82cc2f20ae7c5ac0ad10292fd2656a190d53ac21a2412a47b75d31).